core/new-gui/src/app/workspace/component/operator-panel/operator-panel.component.ts

```diff
@@ -61,6 +79,55 @@ export class OperatorPanelComponent implements OnInit {
     this.operatorSchemaList = operatorMetadata.operators;
     this.groupNamesOrdered = getGroupNamesSorted(operatorMetadata.groups);
     this.operatorGroupMap = getOperatorGroupMap(operatorMetadata);
+
+
+
+    this.filteredOptions = this.myControl.valueChanges
+      .map(value => this._filter(value));
+    }
+
+  private _filter(value: string): string[] {
+
+    type userFriendlyNamesType = {
+      title: string;
+    };
+
+    const userFriendlyNames = this.operatorSchemaList.map(value => value.additionalMetadata.userFriendlyName);
+
+    const new_userFriendlyNames : userFriendlyNamesType[] = []
+    for (let entry of userFriendlyNames) {
+      new_userFriendlyNames.push({'title':entry})
+    }
+
+    const filterValue = value.toLowerCase();
+
+    const options: Fuse.FuseOptions<userFriendlyNamesType> = {
+      location:0,
+      distance:1000,
+      threshold: 0.3,
+      keys: ['title']
+    };
+    const fuse = new Fuse(new_userFriendlyNames,options);
+    const result = fuse.search(filterValue);
+    const final_result = []
+    for (let entry of result) {
+       final_result.push(entry.title);
+    }
+    return final_result;
+  }
+
+  public OnHumanSelected(option: string) {
+    // display the operator on the workflow
+    console.log(option);
+    const currentType = this.operatorSchemaList.filter(
+      schema => {
+        return schema.additionalMetadata.userFriendlyName === option;
+      }
+    ).map(schema => schema.operatorType)[0];
+    console.log(currentType);
+    const selectedOperatorPredicate = this.workflowUtilService.getNewOperatorPredicate(currentType);
+    console.log(selectedOperatorPredicate);
+    this.workflowActionService.addOperator(selectedOperatorPredicate, {x:600, y:399});
```

Comment:
Change the method name to something like `onSearchSelected(operatorName: string)`

Remove the log messages you have for this method.

For the fixed X and Y coordinates when adding a new operator, make them as a readonly variable existing inside this commponent.

Reply:
In addition, add comments to this method as well